docs/io.cozy.salaries.md

```diff
@@ -0,0 +1,48 @@
+[Table of contents](README.md#table-of-contents)
+
+# Salaries doctype
+
+## `io.cozy.salaries`
+
+- `vendor`: {string} - Vendor which issued the salary
+- `date`: {date} - Date the salary was emitted
+- `amount`: {number} - Amount of the bill, __always positive__ even if it is the salary of an employee
+- `payslip`: {string} - The associated file. ex: `io.cozy.files:c43645a93831827c7ec512eac3006e51`
+
+
+### Some more attributes for employee's payslip
+
+The are some more possible attributes for employee salaries.
+
+- `isEmployee`: {boolean} - Indicate if the bill represents a refund/reimbursement (defaults to false)
```

Comment:
Let's call it isReimbursement if it is meant to represent that.

Reply:
This is not really a refound.

I think we should keep `isEmployee` and change the description to `Indicate if this salary is paid to an employee of the owner of the cozy. If false, this is a salary received by the owner.`

Reply:
Yes, my bad

Reply:
Would it make sense to model this as a relationship, so some Pajemploi app could for example introduce a *childminder* doctype? Or is it preferable to put the relationship on the *childminder* side? Or maybe those `isEmployee` & `beneficiary` attributes are required by the *Banks* app for some purpose? (in which case it could be documented here)

Please note I'm not comfortable with document relationships yet, so feel free to correct me in case I'm wrong.